### PR TITLE
Search user in ldap groups

### PR DESCRIPTION
**Pull request by AlexLov**
This PR is related to #2538 and does exactly what requested.

I have very custom LDAP scheme and there is no `MemberOf` attribute for user but `member` attribute in each group with list of user's DN.

### CODE REVIEW

File: pkg/login/ldap.go [hunk 1]
```diff
@@ -54,6 +54,17 @@ func (a *ldapAuther) login(query *LoginUserQuery) error {
 	if ldapUser, err := a.searchForUser(query.Username); err != nil {
 		return err
 	} else {
+
+		for _, ldapGroup := range a.server.LdapGroups {
```
Comment (torkelo) [inline pkg/login/ldap.go]: should there not a a condition for this group lookup? For all ldap installs that have memberOf

Comment (AlexLov) [inline pkg/login/ldap.go]: I generally do not like to add more options to a config, but you are right there should be an option to switch between different LDAP schemes. 
Something like `search_in_groups = true/false`. What do you think, is this name understandable?

**Comment by abligh** [comment]
I did what I think is a more general version of this here: https://github.com/grafana/grafana/pull/2928 (third commit). This allows the group search to be specified and isn't reliant on hardcoding the (replacement) member attribute. I think this should work in your case as well as the standard POSIX schema but would be interested in  your views.

**Comment by torkelo** [comment]
fixed by https://github.com/grafana/grafana/pull/2928

**Pull request closed without merging**